pywatts/modules/postprocessing/merger.py

```diff
@@ -0,0 +1,95 @@
+from typing import Dict, Union
+
+import numpy as np
+import xarray as xr
+
+from pywatts.core.base import BaseTransformer
+from pywatts.core.exceptions import WrongParameterException
+from pywatts.utils._xarray_time_series_utils import numpy_to_xarray
+
+
+class Merger(BaseTransformer):
```

Comment:
Are there any additional requirements on the input time series? Since it's a pyWATTS module, I assume that all dimensions of the time series have the same length. Is that right?

Reply:
It is ensured by xarray that for each time step the second dimension has the same length.